internal/controller/dpuoperatorconfig_controller.go

```diff
@@ -18,24 +18,48 @@ package controller
 
 import (
 	"context"
+	"embed"
 	"fmt"
-	"os"
+	"path/filepath"
+	"sort"
+	"strings"
 
+	"github.com/go-logr/logr"
 	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/apply"
-	"github.com/openshift/cluster-network-operator/pkg/render"
 	configv1 "github.com/openshift/dpu-operator/api/v1"
-	appsv1 "k8s.io/api/apps/v1"
+	"github.com/openshift/dpu-operator/pkgs/render"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/yaml"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+//go:embed bindata/*
```

Comment:
I guess this can't reference the root directory?

Reply:
Correct. It works relatively to the `.go` file, which is nice because as long as the `.go` file lives in the same directory as the embedded files, this will work.